video_xblock/tests/unit/test_backends.py

```diff
@@ -282,6 +283,51 @@ def test_authenticate_api(self, backend, token, auth_mock):
             expected_error = mock.expected_value[-1]
             self.assertIn(expected_error, error)
 
+    def split_transcript_data(self, transcripts):
+        """
+        Splits transcript data to:
```

Comment:
Please split the docstring title with the body by the empty line or add it if it is missed.

Reply:
done